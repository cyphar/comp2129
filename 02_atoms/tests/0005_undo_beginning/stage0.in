START 2 5 5
PLACE 0 0
PLACE 1 1
PLACE 2 2
UNDO
UNDO
UNDO
UNDO
UNDO
