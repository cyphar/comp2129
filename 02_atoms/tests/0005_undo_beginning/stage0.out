Game Ready
Red's Turn

Green's Turn

Red's Turn

Green's Turn

Red's Turn

Green's Turn

Red's Turn

Cannot Undo

Cannot Undo

